pageserver/src/tenant/timeline.rs

```diff
@@ -1898,6 +1924,11 @@ impl Timeline {
                     // about corner cases like s3 suddenly hanging up?
                     self.remote_client.shutdown().await;
                 }
+                Err(FlushLayerError::Cancelled) => {
+                    // this is likely the second shutdown, ignore silently.
+                    // TODO: this can be removed once https://github.com/neondatabase/neon/issues/5080
```

Comment:
5080 is marked as closed. Should it be reopened?

Reply:
It is one of these class issues... I made a note there. I think it doesn't really matter what are we targeting, as long as we are consistent what issue are we targeting.